app/screens/settings/notification_settings/notification_settings.js

```diff
@@ -127,6 +127,22 @@ export default class NotificationSettings extends PureComponent {
         });
     };
 
+    saveAutoResponder = (notifyProps) => {
+        const {intl} = this.context;
+
+        if (!notifyProps.auto_responder_message || notifyProps.auto_responder_message === '') {
```

Comment:
`!notifyProps.auto_responder_message `

checks if the auto_responder_message is undefined and empty.